apps/docs/src/components/Hero/Reveal.svelte

```diff
@@ -1,26 +1,29 @@
 <script lang="ts">
-	import { clamp, mapLinear } from 'three/src/math/MathUtils'
+  import { clamp, mapLinear } from 'three/src/math/MathUtils'
 
-	export let progress: number
+  export let progress: number
 
-	export let from: number = 0
-	export let to: number = 1
+  export let from: number = 0
+  export let to: number = 1
 
-	$: p = clamp(mapLinear(progress, from, to, 0, 1), 0, 1)
+  $: p = clamp(mapLinear(progress, from, to, 0, 1), 0, 1)
 </script>
 
-<div class="reveal" style="--progress: {p};">
-	<slot />
+<div
+  class="reveal"
+  style="--progress: {p};"
+>
```

Comment:
small thing, I like the ending brackets `>` to not be on a line by themselves. though I think some folks like it this way instead 🤷

this way is the default but not sure why

Reply:
I like having them on a single line, makes indentation clear.

Reply:
all good